plugin/src/helpers/edge.ts

```diff
@@ -30,45 +32,42 @@ const loadMiddlewareManifest = (netlifyConfig: NetlifyConfig): Promise<Middlewar
 /**
  * Convert the Next middleware name into a valid Edge Function name
  */
-const sanitizeName = (name: string) => `next${name === '/' ? '_index' : name.replace(/\W/g, '_')}`
+const sanitizeName = (name: string) => `next_${name.replace(/\W/g, '_')}`
```

Comment:
I imagine it's a change in Next.js, but I guess the name is never `/` anymore, hence the change?